sdk/communication/Azure.Communication.Sms/api/Azure.Communication.Sms.net8.0.cs

```diff
@@ -2,9 +2,9 @@ namespace Azure.Communication.Sms
 {
     public partial class MessagingConnectOptions
     {
-        public MessagingConnectOptions(string apiKey, string partner) { }
-        public string ApiKey { get { throw null; } }
+        public MessagingConnectOptions(string partner, object partnerParams) { }
```

Comment:
Can you regenerate the api files? These aren't all changes

Reply:
done